Header header
uint16 shooterHeat0
uint16 shooterHeat1